File: 백준/Bronze/1546. 평균/평균.py
import sys
n = int(sys.stdin.readline())
score = sys.stdin.readline().split()
score = list(map(int, score))

lst = []
max_score = max(score)
for i in score:
    new = i / max_score * 100
    lst.append(new)

print(sum(lst) / n)

File: 백준/Bronze/1546. 평균/평균.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        int N = sc.nextInt();
        int[] score = new int[N];
        float max = 0;

        // 배열에 넣기
        for (int i = 0; i < N; i++) {
            int n = sc.nextInt();
            score[i] = n;
            if (n > max)
                max = n;
        }

        float sum = 0;
        for (int i = 0; i < N; i++) {
            sum += (score[i] / max) * 100;
        }

        float result = sum / N;

        System.out.println(result);
    }
}